.github/workflows/cypress_test.yml

```diff
@@ -0,0 +1,15 @@
+name: Cypress Tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    name: Headless Tests
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      # Install npm dependencies, cache them correctly
```

Comment:
you missed to install dependencies:
`      - name: Install dependencies
        run: npm ci`